powersimdata/network/usa_tamu/constants/zones.py

```diff
@@ -2,6 +2,28 @@
 
 import pandas as pd
 
+__all__ = [
+    "interconnect_combinations",
```

Comment:
We could pre-run this through `sorted` so it's slightly easier to read

Reply:
Done with other simplifications